### PR TITLE
Enable findbugs

### PR DESCRIPTION
**Pull request by thetric**
Enables FindBugs and fixes some warnings

### CODE REVIEW

File: demo/src/main/java/demos/components/ListViewDemo.java [hunk 1]
```diff
@@ -39,13 +42,13 @@ public void start(Stage stage) throws Exception {
         JFXButton buttonExpand = new JFXButton("EXPAND");
         buttonExpand.setOnMouseClicked(e -> {
             list.depthProperty().set(1);
-            list.setExpanded(true);
+            list.setExpanded(TRUE);
```
Comment (skmedix) [inline demo/src/main/java/demos/components/ListViewDemo.java]: But why?

Comment (thetric) [inline demo/src/main/java/demos/components/ListViewDemo.java]: I guess I should disable the corresponding FB rule as it's quite annoying

**Comment by thetric** [comment]
@skmedix I have reverted the lines you commented to the original ones and disabled the stupid FB rule

File: jfoenix/src/main/java/com/jfoenix/controls/JFXComboBox.java [hunk 2]
```diff
@@ -42,6 +42,8 @@
 import java.util.Collections;
 import java.util.List;
 
+import static java.lang.Boolean.FALSE;
```
Comment (skmedix) [inline jfoenix/src/main/java/com/jfoenix/controls/JFXComboBox.java]: One more ;)

**Comment by thetric** [comment]
@skmedix done :sweat_smile:

**Comment by skmedix** [comment]
Looks great! Dzięki 😉

**Comment by jfoenixadmin** [comment]
Looks Great, Thanks for the huge contributions 👍

**Pull request merged**